src/styles/GlobalStyles.ts

```diff
@@ -0,0 +1,21 @@
+import { createGlobalStyle } from 'styled-components';
+import reset from 'styled-reset';
+
+const GlobalStyles = createGlobalStyle` 
+  ${reset} // styled-reset이라는 패키지를 설치하여 reset 설정
+
+  *{
+    box-sizing: border-box;
+  }
+
+  // ThemeProvider 내부에 선언했기에, 테마 가져다 쓸 수 있음.
+  body{
```

Comment:
```suggestion
  body {
```
이부분은 lint나 prettier설정 가능한지 알아보겠습니다.